core/trino-main/src/test/java/io/trino/server/TestSessionPropertyDefaults.java

```diff
@@ -41,27 +45,33 @@
     public void testApplyDefaultProperties()
     {
         SessionPropertyDefaults sessionPropertyDefaults = new SessionPropertyDefaults(TEST_NODE_INFO, new AllowAllAccessControlManager());
+
+        SessionPropertyManager sessionPropertyManager = new SessionPropertyManager();
```

Comment:
nit: test cleanups seem unrelated. Maybe be extracted as separate commit.

Reply:
Unfortunately those changes are required here. We need to use real properties from now on. As we started validating them when setting defaults. And as now they are not self explanatory strings I needed to add comments.